src/test/index.js

```diff
@@ -4,4 +4,5 @@ import './update-features-with-query';
 import './with-features';
 import './configure-feature';
 import './is-feature-included';
-import './features';
```

Comment:
Not sure if you really meant to remove `./features` here, but it will get added back in from the later PRs.